### ISSUE TITLE
make "Solute Amount" display more prominent

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/beers-law-lab#160]
Splitting up feature requests that @kathy-phet made in https://github.com/phetsims/beers-law-lab/issues/159:

> Our colleagues are happy with the recent features added and will use the current version for the study. They had 2 suggestions for us to consider down the road. Logging them here:
> 
> 1) The readout for the solute amount could be more prominently displayed, perhaps in a box with a colored background. 
> ...
> These are considerations if broader use at the middle school level is desired.

**Comment by pixelzoom** [comment]
The "Solute Amount" feature was implemented in https://github.com/phetsims/beers-law-lab/issues/148.  At the time, we considered several different presentation, summarized here:

(1) A panel around the feature. This was deemed too heavy-handed. 

(2) Bold font. Tried this and it looked inconsistent and unnecessarily "weighted". So we decided to use the same font that is used for all other labels and values.
